data_safe_haven/resources/workspace/ansible/desired_state.yaml

```diff
@@ -30,6 +30,13 @@
         state: present
       loop: "{{ snap_packages }}"
 
+    # https://ubuntu.com/server/docs/nvidia-drivers-installation#installing-the-drivers-on-servers-andor-for-computing-purposes
+    - name: Use ubuntu-drivers to install Nvidia drivers  # noqa: no-handler
+      tags: nvidia
+      ansible.builtin.command:
+        cmd: ubuntu-drivers install --gpgpu
+        creates: /usr/bin/nvidia-smi
```

Comment:
Isn't this going to attempt an install on a non-GPU machine now? What happens then?

Reply:
Tested that, it reports "nothing to install"